src/material/expansion/expansion.spec.ts

```diff
@@ -223,6 +223,24 @@ describe('MatExpansionPanel', () => {
     expect(document.activeElement).toBe(header, 'Expected header to be focused.');
   }));
 
+  it('should not change focus origin if origin not specified', fakeAsync(() => {
+    const fixture = TestBed.createComponent(PanelWithContent);
+    fixture.componentInstance.expanded = true;
+    fixture.detectChanges();
+    tick(250);
+
+    const header = fixture.debugElement.query(By.css('mat-expansion-panel-header'))!;
+    const headerInstance = header.componentInstance;
+
+    headerInstance.focus('mouse');
```

Comment:
This isn't the best test in my opinion. I attempted to focus the button with origin `mouse` but this requires making this button a `mat-button` in the component template below.

```
    const button = fixture.debugElement.query(By.css('button'))!;
    const buttonInstance = button.componentInstance as MatButton;

    buttonInstance.focus('mouse');
```

This requires importing MatButton as well, which I was having trouble doing and was not sure if we should avoid importing other modules. If this is a better, however, can change it to this approach (assuming it works).

Reply:
I think the way you have it here is fine